src/Makefile.am

```diff
@@ -60,6 +60,7 @@ decode-teredo.c decode-teredo.h \
 decode-udp.c decode-udp.h \
 decode-vlan.c decode-vlan.h \
 decode-mpls.c decode-mpls.h \
+decode-template.c decode-template.h \
```

Comment:
Guess we could hide it behind a configure option, but that would complicate the code search and replace a bit since we'd have to rename/remove #ifdefs too.